modules/@apostrophecms/submitted-draft/ui/apos/components/AposSubmittedDraftAdminBarIcon.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <span v-if="count > 0" class="apos-submitted-draft-admin-bar-container">
+    <AposIndicator
+      icon="tray-full-icon"
+      class="apos-input-icon apos-submitted-draft-admin-bar-icon"
+      :icon-size="size"
+    />
+    <span v-if="canPublish" class="apos-submitted-draft-admin-bar-counter">
+      {{ count }}
+    </span>
+  </span>
+</template>
+<script>
+export default {
+  name: 'AposSubmittedDraftAdminBarIcon',
```

Comment:
Shorten to `AposSubmittedDraftIcon`? The context of the admin bar doesn't seem critical.